fleet_vehicle_service_kanban/views/fleet_vehicle_log_services_views.xml

```diff
@@ -63,7 +63,7 @@
                 <attribute name="default_group_by">stage_id</attribute>
 
                 <xpath
-                    expr="//div[contains(@class, 'oe_kanban_details')]"
+                    expr="//div[contains(@hasclass, 'oe_kanban_details')]"
```

Comment:
Is this working?

To my knowledge `hasclass` is used like this `expr="//div[hasclass('oe_kanban_details')]`

Otherwise, the way I understand it, you're targetting elements that have `oe_kanban_details` in their `hasclass` attribute (which, normally, shouldn't match anything?)

(same comment goes to the other case in this commit)

Reply:
Certainly, I conducted some tests and it wasn't working properly as you pointed out. I have made the necessary changes accordingly. Thank you for your feedback and review.